daca.md

```diff
@@ -0,0 +1,60 @@
+VeXXaN Digital/Analog Control Adapter
+----
+
+```
+              __  ____  __ __   _    __
+    /\   /\___\ \/ /\ \/ /__ \ / \  /_/ 
+    \ \ / / _ \\  /  \  // _  / ^ \     
+  __ \ V /  __//  \  /  \\___/ / \ \    
+ /_/  \_/ \___/_/\_\/_/\_\   \/   \/   
+
+```
+
+
+|     Item       |   Value    |   Comment
+| -------------: | ---------- | ----------------
+|    Vendor code | 0x621CAF3A | VeXXaN Performance Peripherals
+|      Device ID | 0x31E1DACA | Digital/Analog Control Adapter 
+|    Device type | 0x31E1     | Human Interface Device, Game Controller, Can Generate Interrupts
+|        Version | 0x0001     | Version 1
+
+Device Description
+
+VeXXaN Performance Peripherals(tm) is proud to announce the only control adapter you will ever need!  With the ability to connect up to 4(!) 8-bit joysticks and 16 buttons using a standard DB25 connection, the days of direct attached game controllers ARE! IN! THE! PAST!  I personally guarantee that you will find more uses for this adapter than you ever thought possible! Remote control vehicles? Done! Arcade Machines? Sure! Spacecraft controls? You Betcha!*  Order within the next 5 Minutes and we'll throw in our patented No-Slip rubberized joystick base and a full set of our low-travel digital buttons! (just pay seperate processing and handling)** And, Best of all, If you're not 100,000% satisfied with your brand new DACA, simply return it for a full refund.***
```

Comment:
I can see where the confusion can come from here.  The spec lays out that there are 4 analog axis available, and I feel that it fits the "Infomercial" tone of the description to refer to an analog axis as a "Joystick", so I will add another disclaimer to the description as a means to explain it.